nucypher/characters/lawful.py

```diff
@@ -318,8 +316,17 @@ def grant():
 
         @alice_control.route('/revoke/<policy_encrypting_key>', methods=['DELETE'])
         def revoke(policy_encrypting_key):
+            policy = drone_alice.active_policies[policy_encrypting_key]
```

Comment:
I've just realized that `active_policies` are not persisted anywhere, right? So if Alice tries to revoke at a latter time, this won't work.

Reply:
It's what we've meant to do for the time being, actually. Enabling persistent policies will encompass a larger PR.

Reply:
I see, that's fine. Same as above: does this deserve an issue?

Reply:
Probably does deserve an issue, yeah. We're all well aware of it and it would be nice to talk about it in an issue.